ReactiveCocoa/Swift/Property.swift

```diff
@@ -60,6 +60,12 @@ public struct AnyProperty<Value>: PropertyType {
 	}
 }
 
+extension PropertyType {
+	public func map<U>(transform: Value -> U) -> AnyProperty<U> {
+		return AnyProperty(initialValue: transform(value), signal: signal.map(transform))
```

Comment:
I'm not sure why this would not be thread safe. What do you mean by "skip a value"?